locals.js

```diff
@@ -8488,6 +8522,22 @@ I18N.zh["repository/commit"] = { // 仓库 - 提交页面
             
                 "Reply…": "回复…",
 
+            "commented on": "评论于",
+
+            // 隐藏评论
+            "Choose a reason for hiding this comment": "选择隐藏此评论原因",
+            "The reason will be displayed to describe this comment to others.": "将显示原因，以便向其他人描述此评论。",
+                "Choose a reason for hiding this comment": "选择隐藏此评论原因",
+                "Unhide": "取消隐藏",
+                "comment": "评论",
+                "Choose a reason": "选择原因",
+                    "Abuse": "滥用",
+                    "Spam": "垃圾信息",
+                    "Off Topic": "离题",
```

Comment:
```suggestion
                    "Off Topic": "偏离主题",
```
此处参考了官方app，另外其他几处也一并修改一下哈